system-tests/test/record_spec.js

```diff
@@ -2308,6 +2308,31 @@ describe('e2e record', () => {
         })
       })
 
+      describe('when the tab crashes in chrome', () => {
+        enableCaptureProtocol()
+        it('still uploads a test replay', function () {
+          return systemTests.exec(this, {
+            key: 'f858a2bc-b469-4e48-be67-0876339ee7e1',
+            configFile: 'cypress-with-project-id.config.js',
+            browser: 'chrome',
+            spec: 'chrome_tab_crash.cy.js,simple.cy.js',
+            record: true,
+            snapshot: true,
+            expectedExitCode: 1,
+          }).then(() => {
+            const urls = getRequestUrls()
+            const requests = getRequests()
+            const postResultsRequest = requests.find((r) => r.url === `POST /instances/${instanceId}/results`)
+
+            console.log(JSON.stringify(postResultsRequest, null, 2))
+
+            console.log(getRequests())
```

Comment:
Should these be committed?

Reply:
No, and also you found an incomplete test - thanks! I need to add some assertions around what's being logged here